README.md

```diff
@@ -61,50 +61,38 @@ Extension | Contributor | Description
 [touchscreen-buttons](https://github.com/jspsych/jspsych-contrib/blob/main/packages/extension-touchscreen-buttons/README.md) | [Younes Strittmatter](https://github.com/younesStrittmatter) | This extension displays touch buttons that allow the participant to respond to stimuli via a touchscreen on mobile devices. 
 
 ## Guidelines for contributions
+### Contribution requirements
 Contributions to this repository must:
 
 * Work as described
 * Include the complete code for the plugin or extension.
-* Include a `readme.md` file following our [template](readme-template.md).
+* Include a `readme.md` file following our [template]([identical for plugins and extensions](https://github.com/jspsych/jspsych-dev/blob/main/packages/new-plugin/templates/plugin-template-ts/README.md)).
 * Include a `package.json` file.
 
-Optionally, contributions can include:
+Optionally, contributions are encouraged to include:
 
 * A `/docs` directory with documentation matching the template for docs on jspsych.org 
 * An `/examples` directory with a working `.html` demo.
-* A test suite following the testing framework in our `-ts` templates.
+* A test suite following the testing framework in our `-ts` [templates]([plugin test template](https://github.com/jspsych/jspsych-dev/blob/main/packages/new-plugin/templates/plugin-template-ts/src/index.spec.ts); [extension test template](https://github.com/jspsych/jspsych-dev/blob/main/packages/new-extension/templates/extension-template-ts/src/index.spec.ts)).
 
+### To contribute to this repository, follow these steps:
```

Comment:
To contribute a new plugin or extension to this repository...

(thinking some contributions are just edits and don't need tools)